PROJECT

```diff
@@ -53,4 +53,12 @@ resources:
   kind: KueueConfiguration
   path: sigs.k8s.io/kueue/apis/config/v1alpha1
   version: v1alpha1
+- api:
```

Comment:
It sounds like this and the rest of API files shouldn't be in this PR?

Reply:
That is in fact part of #1045, this pr is still draft